temboardagent/plugins/dashboard/metrics.py

```diff
@@ -203,22 +203,20 @@ def get_memory_usage(self,):
             return self._get_memory_usage_linux()
 
     def get_stat_db(self,):
-        query = """
-SELECT
-  count(datid) as databases,
-  pg_size_pretty(sum(pg_database_size(
-    pg_database.datname))::bigint) as total_size,
-  to_char(now(),'HH24:MI') as time,
-  sum(xact_commit)::BIGINT as total_commit,
-  sum(xact_rollback)::BIGINT as total_rollback
-FROM
-  pg_database
-  JOIN pg_stat_database ON (pg_database.oid = pg_stat_database.datid)
-WHERE
-  datistemplate = 'f'
-        """
-        self.conn.execute(query)
-        row = list(self.conn.get_rows())[0]
+        row = self.conn.queryone("""\
+        SELECT
+        count(datid) as databases,
+        pg_size_pretty(sum(pg_database_size(
+            pg_database.datname))::bigint) as total_size,
+        to_char(now(),'HH24:MI') as time,
+        sum(xact_commit)::BIGINT as total_commit,
+        sum(xact_rollback)::BIGINT as total_rollback
+        FROM
+        pg_database
+        JOIN pg_stat_database ON (pg_database.oid = pg_stat_database.datid)
+        WHERE
+        datistemplate = 'f'
+        """)
```

Comment:
Is it possible to make it more readable? ie. avoid linebreak after FROM and WHERE, and indent fields.

Reply:
Fixed in #439